### PR TITLE
Align api definition

### PR DESCRIPTION
**Pull request by angelbarrera92**
This PR contains the alignment with the API definition created days ago it also includes the latest changes in the health check golang template.

### CODE REVIEW

File: pkg/jobs/jobs.go [hunk 1]
```diff
@@ -6,18 +6,43 @@ package jobs
...
+	KfipBaseLabel = "kfip.sighup.io"
+
+	// AppBaseLabel use this label as base of other labels like name or component.
+	AppBaseLabel = "app.kubernetes.io"
 )
 
 // GetJobs returns a list of Jobs from a specific namespace, or all the jobs in a cluster if namespace is "".
-func GetJobs(ctx context.Context, kc *kube.KubernetesClient, namespace string) ([]batchv1.Job, error) {
-	jobList, err := kc.Client.BatchV1().Jobs(namespace).List(ctx, v1.ListOptions{})
+func GetJobs(ctx context.Context, kc *kube.KubernetesClient, group string, target string) ([]batchv1.Job, error) {
```
Comment (nandajavarma) [inline pkg/jobs/jobs.go:28]: How about we send in the jobLabels as a parameter? Just to make the GetJobs as generic as possible?

File: internal/server/responses.go [hunk 2]
```diff
@@ -5,47 +5,134 @@
...
 
 // Job represents the json object of the jobs attribute in the main response.
 type Job struct {
-	Name           string `json:"name"`
-	Namespace      string `json:"namespace"`
-	Succeeded      bool   `json:"succeeded"`
-	StartTime      string `json:"startTime"`
-	CompletionTime string `json:"completionTime"`
+	Group       string `json:"group"`
+	Target      string `json:"target"`
```
Comment (nandajavarma) [inline internal/server/responses.go:26]: More of a query than a comment. I think something I would like to know from the job status API would be what is the target resource that is being monitored. As in, if it is a service, deployment, pod or whatever and along with that the name of the resource itself(service name, deployment name, etc.). Would you say we will get it from group + target combination?

File: internal/server/server.go [hunk 3]
```diff
@@ -35,27 +37,52 @@ func Setup(cfg *config.JobStatusAPIConfig) *gin.Engine {
 			c.String(http.StatusOK, "OK\n")
 		}
 	})
-	r.GET("/jobs", func(c *gin.Context) { // Get all jobs in the cluster. Requires clusterRoleBinding
-		jobs, err := jobs.GetJobs(c, &cfg.KubeClient, "")
+	r.GET("/group/:group", func(c *gin.Context) { // Get all jobs in a namespace. Requires roleBinding
```
Comment (nandajavarma) [inline internal/server/server.go:40]: Should we also add an endpoint to get some logs of the jobs or the reason for the failure of a job? This need not even be stored as a part of the struct or anything just to pull realtime if the request is made.

File: pkg/jobs/jobs.go (hunk 1 above)
Comment (angelbarrera92) [reply pkg/jobs/jobs.go:28]: Makes a lot of sense, implementing!

File: internal/server/server.go (hunk 3 above)
Comment (angelbarrera92) [reply internal/server/server.go]: I think we can make it happens on upcoming iterations

File: pkg/jobs/jobs.go (hunk 1 above)
Comment (angelbarrera92) [reply pkg/jobs/jobs.go]: Done!

**Pull request merged**